meta/serialize.pm

```diff
@@ -1093,6 +1094,7 @@ sub EmitDeserializeFooter
         WriteSource "{";
         WriteSource "SAI_META_LOG_WARN(\"nothing was deserialized for '$name', bad condition?\");";
         # WriteSource "return SAI_SERIALIZE_ERROR;";
+        WriteSource "return SAI_SERIALIZE_ERROR;" if $name eq "sai_attribute_value_t";
```

Comment:
why this is added?

Reply:
This is for giving the caller a chance to serialize the attribute value as array of uint8 if nothing is deserialized for it.